pages/home_page.py

```diff
@@ -0,0 +1,129 @@
+from playwright.sync_api import Page, expect
+
+
+class HomePage:
+    def __init__(self, page: Page):
+        self.page = page
+        self.signup_login_button = self.page.locator('a[href="/login"]')
+        self.accept_cookies_button = self.page.locator('button.fc-cta-consent')
+        self.logged_in_user = self.page.locator("li:has-text('Logged in as') b")
+        self.delete_account = self.page.locator("li:has-text('Delete Account')")
+        self.logout = self.page.locator("li:has-text('Logout')")
+        self.contact_us = self.page.locator("header li:nth-child(8)")
+        self.test_cases = self.page.locator('header li:nth-child(5)')
+        self.homepage_logo = page.locator('img[src*="logo.png"]')
+        self.homepage_logo = page.locator('img[src*="logo.png"]')
+        self.subscription_text = page.locator('h2', has_text="Subscription")
+        self.subscription_email_input = page.locator('#susbscribe_email')
+        self.subscription_button = page.locator('#subscribe')
+        self.subscription_success_message = self.page.locator('#success-subscribe .alert-success')
+        self.cart_button = page.locator('header li:nth-child(3)')
+        self.products_button = page.locator('a[href="/products"]')
+        self.view_product_button = page.locator('.features_items .product-image-wrapper .choose a', has_text="View Product")
+        self.categories_section = page.locator('.left-sidebar h2', has_text="Category")
+        self.women_category = page.locator('a[data-toggle="collapse"][data-parent="#accordian"][href="#Women"]')
+        self.men_category = page.locator('a[data-toggle="collapse"][data-parent="#accordian"][href="#Men"]')
+        self.recommended_items_section = page.locator('h2', has_text="Recommended items")
+        self.recommended_add_to_cart_button = page.locator('.recommended_items .btn.btn-default.add-to-cart')
+        self.view_cart_button = page.locator('a[href="/view_cart"]', has_text="View cart")
+        self.subscription_section = page.locator('h2', has_text="Subscription")
+        self.scroll_up_button = page.locator('a[id=scrollUp]')
+        self.carousel_elements = page.locator('.carousel-inner .item h2', has_text="Full-Fledged practice website for Automation Engineers")
+
+
+    def goto_home_page(self):
+        self.page.goto("http://automationexercise.com")
+
+    def verify_home_page_visible(self):
+        return self.page.title() == "Automation Exercise"
+
+    def click_signup_login(self):
+        self.signup_login_button.first.click()
+
+    def click_delete_account(self):
+        self.delete_account.click()
+
+    def click_logout(self):
+        self.logout.click()
+
+    def click_cart_button(self):
+        self.cart_button.click()
+
```

Comment:
@karenn32 There is no reason for the empty space here, please, remove it. Try to make the code as clean as possible.